source/manual/alerts/recreate-a-mongo-instance-in-aws.html.md

```diff
@@ -37,17 +37,17 @@ In this example we are looking at the `integration` environment.
    - https://github.com/alphagov/govuk-aws-data
    - https://github.com/alphagov/govuk-secrets
 
-1. It is very useful to have access to the AWS console, to observer the changes
+2. It is very useful to have access to the AWS console, to observe the changes
    as they happen.
 
-1. You can now execute a `terraform plan` to view the actual module names.
+3. You can now execute a `terraform plan` to view the actual module names.
 
    ```bash
    $ cd govuk-aws
    $ tools/build-terraform-project.sh -c plan  -p app-mongo -s blue -d data -e integration # Please check the script to identify the keys
    ```
 
-1. The out-put, will show you the components and how they are named by `terraform`.
+4. The out-put, will show you the components and how they are named by `terraform`.
```

Comment:
```suggestion
4. The output, will show you the components and how they are named by `terraform`.
```

Reply:
Good spot. I've just pushed your correction, could you take another look?